src/utils/StringUtils.js

```diff
@@ -113,6 +113,16 @@ define(function (require, exports, module) {
     }
     
     /**
+     * Returns true if the given string starts with the given prefix.
+     * @param   {String} str
+     * @param   {String} prefix
+     * @returns {Boolean}
+     */
+    function startsWith(str, prefix) {
+        return str.slice(0, prefix.length) === prefix;
+    }
```

Comment:
Since ends with is part of string utils, i think it would be ok to have startsWith also here for consistancy.